change/@microsoft-fast-components-9902ce21-7d35-46de-8c68-28f33a5e69de.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Is this an example or a test? If not, I think we'll want this to be "none"

Reply:
Just for testing purpose

Reply:
Cool - I'm going to "hold" while we test just to prevent this from being merged *with this in*. I think once we can confirm this is working as expected we'll need to publish w/ Lerna, merge this immediately after, and that will start enforcing these files. 👍